src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -200,8 +200,45 @@ collect_mf(_Registry, Callback) ->
          Size = ets:info(Table, size),
          mf_totals(Callback, Name, Type, Help, Size)
      end || {Table, Name, Type, Help} <- ?TOTALS],
+    add_metric_family(build_info(), Callback),
+    add_metric_family(identity_info(), Callback),
     ok.
 
+build_info() ->
+    {ok, PrometheusPluginVersion} = application:get_key(rabbitmq_prometheus, vsn),
+    {ok, PrometheusClientVersion} = application:get_key(prometheus, vsn),
+    {
+        build_info,
+        untyped,
+        "RabbitMQ & Erlang/OTP version info",
+        [{
+            [
+                {rabbitmq_version, rabbit_misc:version()},
+                {prometheus_plugin_version, PrometheusPluginVersion},
+                {prometheus_client_version, PrometheusClientVersion},
+                {erlang_version, rabbit_misc:otp_release()}
+            ],
+            1
+        }]
+    }.
+
+identity_info() ->
+    {
+        identity_info,
+        untyped,
+        "Node & cluster identity info",
+        [{
+            [
+                {node, node()},
```

Comment:
I'd be careful with labels called node and cluster, there's a good chance that the user will already have them as target labels. rmq_node and rmq_cluster maybe?

Reply:
`node` and `cluster` are first class RabbitMQ concepts. Maybe `node_name` and `cluster_name`...

Reply:
If there's potential for collision or confusion in larger systems that use Prometheus, `rabbitmq_node_name` and `rabbitmq_cluster_name` is what I'd use.

Reply:
The _name is redundant there.

Reply:
Fair enough.

Reply:
Using `rabbitmq_node` & `rabbitmq_cluster` https://github.com/rabbitmq/rabbitmq-prometheus/pull/4/commits/5370cabe2c4ddc2530903d8795b8212740b6de9e#diff-fa0286c717bf744481489229d1e4ab5e